src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -292,7 +292,7 @@ else if( channel == AEApi.instance().storage().getStorageChannel( IFluidStorageC
 		final MEInventoryHandler<T> ih = new MEInventoryHandler<T>( h, h.getChannel() );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2293)